css/navbar.css

```diff
@@ -0,0 +1,112 @@
+.header {
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+  height: 4rem;
+  display: flex;
+  align-items: center;
+  position: relative;
+  justify-content: space-between;
+  border-bottom-right-radius: 2rem;
+}
+
+.header-brand {
+  padding: 2rem;
+  font-size: 1.4rem;
+  font-weight: 600;
+}
+
+.navbar {
+  background-color: var(--primary-color);
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 30vw;
+  height: 100%;
+  transition: all 0.7s ease;
+  -webkit-transition: all 0.7s ease;
+  -moz-transition: all 0.7s ease;
+  -ms-transition: all 0.7s ease;
+  -o-transition: all 0.7s ease;
+  z-index: 2;
+}
+
+.navbar-show {
+  width: 30vw;
+  display: block;
+  transition: all 0.7s ease;
+  -webkit-transition: all 0.7s ease;
+  -moz-transition: all 0.7s ease;
+  -ms-transition: all 0.7s ease;
+  -o-transition: all 0.7s ease;
```

Comment:
**Fix:** Just a suggestion here that as all the browsers have full support for transitions, hence you can avoid giving it explicitly for all the browsers.

Reply:
It gets added automatically by my VSCode due to the extension. I guess I'll just keep it for now!